js/repl.ts

```diff
@@ -53,81 +54,65 @@ export async function readline(rid: number, prompt: string): Promise<string> {
 
 // @internal
 export async function replLoop(): Promise<void> {
-  window.deno = deno; // FIXME use a new scope (rather than window).
+  window.deno = deno;
 
   const historyFile = "deno_history.txt";
   const rid = startRepl(historyFile);
 
-  let code = "";
   while (true) {
+    let code = "";
     try {
-      code = await readBlock(rid, "> ", "  ");
+      code = await readline(rid, "> ");
+      if (!code) {
+        continue;
+      } else if (code.trim() === ".exit") {
+        break;
+      }
+      while (!evaluate(code)) {
+        code += "\n";
+        code += await readline(rid, "... ");
```

Comment:
nit: use "  " (two spaces) here so that indentation is preserved and code is copy paste-able.
This is in line with Chrome, and waay more useful that Node's behavior.

Reply:
Justified.
The only concern I had before was if `"  "` is clear enough that Deno is waiting for user input instead of just hanging/stuck/running on some slow blocking code. But yeah it's probably less important compared to the ease of copy & pasting...

Reply:
An alternative is to use some characters that also does not affect code running for copy-pasting:
```
> {
/**/ a: 1
/**/ }
{ a: 1 }
```
We can make `/**/` grayish to show they are not from user input and less obtrusive.

On the other hand, we probably could just imitate chrome even closer:
<img width="98" alt="screen shot 2018-12-27 at 3 02 39 pm" src="https://user-images.githubusercontent.com/15007517/50496938-80d4c080-09e8-11e9-9b8e-e484a8cffa5a.png">
```
> {
    a: 1
}
< { a: 1 }
```

Reply:
I think I like the `>` `<` combo most.